clusterloader2/testing/storage/pod-startup/ephemeral-volumes/pod.yaml

```diff
@@ -0,0 +1,49 @@
+{{$volumeType := .VolumeType}}
+{{$index := .Index}}
+{{$appName := .AppName}}
+{{$volumesPerPod := .VolumesPerPod}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{.Name}}
+  labels:
+    app: {{.Name}}
+    group: {{.Group}}
+spec:
+  containers:
+  - name: {{.Name}}
+    image: k8s.gcr.io/pause:3.1
+    imagePullPolicy: IfNotPresent
+    ports:
+    - containerPort: 80
+      name: {{.Name}}
+    volumeMounts:
+    {{ range $volumeIndex, $vol := Seq .VolumesPerPod }}
+    - name: vol-{{$volumeIndex}}
+      mountPath: /usr/share/{{$volumeIndex}}
+    {{ end }}
+  volumes:
+  {{ range $volumeIndex, $vol := Seq .VolumesPerPod }}
```

Comment:
I'm wondering if this really makes it easier to understand.

I agree that it would result in some duplication of code (or I should say config), but wouldn't having:
pod_with_secrets, pod_with_configmaps, pod_with_emptydirs as separate yamls, be simpler?

Longer-term, I filed: https://github.com/kubernetes/perf-tests/issues/619, but I don't know how easy that is to do.

Reply:
I'll do it, also will move the volume specific yaml files under the specific volume directory